src/main/java/com/exasol/adapter/jdbc/JdbcAdapter.java

```diff
@@ -79,7 +79,7 @@ private SqlDialect createDialect(final Connection connection, final AdapterPrope
 
     @Override
     public DropVirtualSchemaResponse dropVirtualSchema(final ExaMetadata metadata,
-            final DropVirtualSchemaRequest request) {
+                                                       final DropVirtualSchemaRequest request) {
```

Comment:
This indentation does not look right. What happened here?

Reply:
done